deploy/template.yaml

```diff
@@ -1925,6 +1973,28 @@ Resources:
                         - EnvironmentConfiguration
                         - !Ref AWS::AccountId
                         - environment
+        - Statement:
+            - Sid: invokeGetCiFunction
```

Comment:
Sid should be something like `invokeGetContraIndicatorCredentialFunction`

Reply:
done